packages/oui-radio-toggle-group/radio-toggle-group.less

```diff
@@ -5,7 +5,11 @@
 
 .oui-radio-toggle-group {
   display: flex;
-  max-width: 300px;
+  flex-direction: row;
```

Comment:
No impact on removing the `max-width` ?

Reply:
No, the main container is in `inline-block`. But you made think that every form component is by default a `block`.
So to avoid issue with `inline-block` / `block`, I'll update the display `flex` to `inline-flex`. This will have a similar behaviour.